### PR TITLE
feat(suite-native): single fee option ui

### LINKED ISSUE #14333: Send coins - bitcoin-like networks fee settings
# Acceptance criteria:

For other Bitcoin like coins (LTC,  BTC, BTH etc.)  where is only one fee option, change fee selection screen copy according to [this design](https://www.figma.com/design/sqKFZAIvk3zERxiWgthK8K/Send?node-id=2163-18557&t=eU86aJbMDI54zSmX-4).

### PR DESCRIPTION
**Pull request by PeKne**
## Description
- if there is only one fee option possible during send, it is preselected and not interactive. 

## Related Issue

Resolve #14333

## Screenshots:
Screenshot 2024-10-11 at 9 11 54

### CODE REVIEW

File: suite-native/module-send/src/components/FeeValueCard.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    paddingVertical: utils.spacings.sp12,
+}));
+
+export const FeeValueCard = ({ feeValue, networkSymbol }: FeeValueCardProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    return (
+        <Card style={applyStyle(cardStyle)}>
+            <HStack alignItems="center">
+                <Text variant="highlight">Transaction fee</Text>
```
Comment (matejkriz) [inline suite-native/module-send/src/components/FeeValueCard.tsx]: Please make it translatable. 

Also, in the design it is called "Mining fee", but I guess it is old copy? Please clarify in the description or figma comment what should be the current copy.

**Review by matejkriz — COMMENTED** [review]
We've changed the design of BTC fee levels to include fee rate and time estimate, but btc-like coins reflecting older version of fee level options. Fee rate and estimate makes total sense for bitcoin-like coins too. 

We can say it's a nitpick and not a priority now, just make it as a conscious decision.

image
vs 
image

**Comment by PeKne** [comment]
@matejkriz  Good point about the design inconsistency. @shenkys, what is your opinion about it?

**Comment by PeKne** [comment]
@matejkriz the [design](https://www.figma.com/design/sqKFZAIvk3zERxiWgthK8K/Send?node-id=2163-18557&t=GRl0vEicr8Cd9Ip1-4) was unified with the selectable fee options. The last version of the implementation looks like this:
Screenshot 2024-10-16 at 10 40 25

Also, after a discussion with @shenkys, the `Medium` fee option was renamed for all networks to `Normal` to make it consistent with desktop implementation.

Please do the review again 🙏.

**Review by matejkriz — APPROVED** [review]
👍 Looks better, thanks! 

The only nitpick is that other coins have different fee units, but it's not implemented on desktop so let's leave it as it is...

**Pull request merged**